main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.initial
```

Comment:
- [ ] To create a new object in Ruby, you simply need to call the new method on the class. For example, to create a new App object, you would say app = App.new. The initial method is not a valid method on objects in Ruby, so please remove it from your code.